model-with-changes.cto

```diff
@@ -0,0 +1,8 @@
+namespace org.example.basic
```

Comment:
I don't we should add these models here in root directory.

Moreover we might have to also add a version to the model namespace.
`namespace org.example.basic@1.0.0`

Reply:
https://concerto.accordproject.org/docs/design/specification/model-namespaces